setup.py

```diff
@@ -46,4 +46,6 @@
           'Programming Language :: Python :: 3',
           'Topic :: Software Development :: Libraries'],
       requires=['six'],
-      packages=['textfsm'])
+      packages=['textfsm'],
+      setup_requires=['pytest-runner'],
```

Comment:
Since we don't include tests in the distribution - are these two lines required?